as/cNFT/assembly/models/persistent_tokens.ts

```diff
@@ -100,11 +100,22 @@ export class PersistentTokens {
         return token
     }
 
+    /**
+     * @param tokenId: Id of the token to remove
+     * @param accountId: Account of token to remove
+     */
+    remove_token(tokenId: TokenId, accountId: AccountId): void {
+        this._tmap.delete(tokenId);
+
+        this.remove(tokenId, accountId);
+    }
+
     /**
      * @param tokenId: Id of the token to remove
      * @param accountId: Account of token to remove
      */
     remove(tokenId: TokenId, accountId: AccountId): void {
```

Comment:
I think calling this `remove_from_account` and `remove_token` as `remove` makes more sense.

Also, it would be nice to have a comment that describes what they do